src/balancereader/src/main/java/anthos/samples/financedemo/balancereader/BalanceReaderController.java

```diff
@@ -120,10 +121,11 @@ public Long load(String accountId)
             }
         };
         this.cache = CacheBuilder.newBuilder()
-                            .maximumSize(expireSize)
-                            .build(loader);
+            .maximumSize(expireSize)
+            .build(loader);
         // Initialize transaction processor.
         this.ledgerReader = reader;
+        LOGGER.debug("Initialized transaction processor");
```

Comment:
This could probably be bumped up to 'info'

This is a signal into the current state of the program, and it only happens once on startup.  (Log files flooded with INFO messages from repeated operations are hard to debug)